test/system/trader.js

```diff
@@ -131,11 +100,20 @@ class Trader {
       price,
     };
     return this.client.createOrder(order)
-      .then(() => {
-        console.log((side === 'B' ? 'Buy' : 'Sell'), 'Qauntity is', quantity, 'price is' , price, 'order placed', this.config.name);
-      }).catch(err => {
-        console.log(err);
-      });
+      .then(
+        () => {
+          if (options && options.expectFail) {
+            throw new Error('createOrder succeeded when expected to fail');
+          }
+          return  console.log((side === 'B' ? 'Buy' : 'Sell'), 'Quantity is', quantity, 'price is' , price, 'order placed', this.config.name);
```

Comment:
It's not generally a good practice to return the value of console.log. We can achieve the same result by not returning anything.